test/index.ts

```diff
@@ -142,22 +142,23 @@ describe('teeny', () => {
     });
   });
 
-  it('should respect environment variables for proxy config', () => {
-    const envVars = ['http_proxy', 'https_proxy', 'HTTP_PROXY', 'HTTPS_PROXY'];
-    for (const v of envVars) {
+  const envVars = ['http_proxy', 'https_proxy', 'HTTP_PROXY', 'HTTPS_PROXY'];
+  for (const v of envVars) {
+    it('should respect environment variables for proxy config', done => {
```

Comment:
We should change this to something like `should respect the ${v} environment variable`